--- a/find-modules/FindPythonModuleGeneration.cmake     2021-07-07 01:38:31.845926856 +0100
+++ b/find-modules/FindPythonModuleGeneration.cmake     2021-07-07 01:45:40.927272472 +0100
@@ -255,7 +255,7 @@
   message(STATUS "Found ${libclang_LIBRARY}")
 endif()
 
-foreach (pyversion "2" "3")
+foreach (pyversion "3")
   set(GPB_PYTHON_COMMAND ${GPB_PYTHON${pyversion}_COMMAND})
   if (NOT GPB_PYTHON_COMMAND)
     continue()
